plugins/rcpt_to.access.js

```diff
@@ -63,7 +63,7 @@ function _in_whitelist(connection, plugin, address) {
         connection.logdebug(plugin, 'checking ' + address + ' against ' +
             plugin.wl[i]);
 
-        if (plugin.wl[i] === address) {
+        if (plugin.wl[i].toLowerCase() === address.toLowerCase()) {
             return true;
```

Comment:
Instead of lower casing every entry in the whitelist on every connection, performance can be improved by lower casing all the file entries when the file is loaded. That's saves millions and millions of toLowerCase() operations at run time, and you only have to lower the user supplied username.